src/ralph/export_to_ng/publishers.py

```diff
@@ -194,3 +229,5 @@ def sync_virtual_server_to_ralph3(sender, instance=None, created=False, **kwargs
         'parent_id': asset.id if asset else None,
         'custom_fields': _get_custom_fields(instance),
     }
+    data.update(_get_ips_list(instance))
```

Comment:
shouldn't ips be treated just like any other field? if it is on fields list, it will be included in result, otherwise no (in your case ips are always included in result).